ui/src/shared/apis/index.js

```diff
@@ -89,6 +89,18 @@ export const getKapacitors = async source => {
   }
 }
 
+export const deleteKapacitor = async (source, kapacitorID) => {
+  try {
+    return await AJAX({
+      method: 'DELETE',
+      url: `${source.links.kapacitors}/${kapacitorID}`,
```

Comment:
you're right, I can remove `source` from the params and just use the `kapacitor` object